napari/_qt/_qapp_model/_tests/test_layerlist_context_actions.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from napari._app_model import get_app
+from napari._app_model.actions._layerlist_context_actions import (
+    LAYERLIST_CONTEXT_ACTIONS,
+)
+
+
+@pytest.mark.parametrize('layer_action', LAYERLIST_CONTEXT_ACTIONS)
+def test_layer_actions_ctx_menu_execute_command(
+    layer_action, make_napari_viewer
+):
+    app = get_app()
```

Comment:
I think the structure of this test is great. Can you please add a comment in the docstring of the function explaining that we are strictly testing the app-model action dispatch here, and the functions themselves are tested in the other file?